samples/alarmbot-es6-botframework-webchat/src/routes.js

```diff
@@ -0,0 +1,24 @@
+// configure bots routing table
+import {alarms} from "./alarms";
+import {Prompt} from 'botbuilder-prompts';
+
+export function routes(context) {
+    if (context.ifRegExp(/(list|show) alarms/i)) {
```

Comment:
>context.ifRegExp [](start = 8, length = 16)

This was just removed... And it looks like you're working against an older copy of the alarm bot sample.

Reply:
+1 Updated to use the latest alarm bot example